pubspec.yaml

```diff
@@ -41,6 +42,7 @@ dependencies:
   http: ^1.1.0
   intl: ^0.18.0
   json_annotation: ^4.7.0
+  linkify: ^5.0.0
```

Comment:
Why do we need this package since `flutter_linkify` is already handling the job alone it seems?

Reply:
This is cause `flutter_linkify` is depended on this library but it's not showing `UserTagLinkifier` which is needed to link user tags.
[repo link](https://github.com/Cretezy/flutter_linkify/blob/b43e490da124721d003f5f95f000372548db27e3/lib/flutter_linkify.dart#L5)

One way to solve that would be to create our own `Linkify` widget by copying it from `flutter_linkify`